lib/quby/questionnaires/services/definition_validator.rb

```diff
@@ -6,6 +6,7 @@ module Questionnaires
     module Services
       class DefinitionValidator < ActiveModel::Validator
         MAX_KEY_LENGTH  = 19
+        MAX_SHORT_KEY_LENGTH = 8
```

Comment:
Om deze ticket dan toch maar es door te laten gaan, lijkt me een lengte van 4 wel prima, zijn nu al heel meestal 3 of 4 volgens mij. Heb nu niet zoveel zin om uit te zoeken welke keys allemaal geconcateneerd worden in het dagelijks leven, maar ik zie liever een korte score_key dan een korte questionnaire_key.